analysis/bin/main.ml

```diff
@@ -110,6 +110,23 @@ let main () =
         path line col
   in
   match args with
+  | [_; "cache-project"; rootPath] -> (
+    Cfg.useProjectConfigCache := false;
+    let uri = Uri.fromPath rootPath in
+    match Packages.getPackage ~uri with
+    | Some package -> Cache.cacheProject package
+    | None -> print_endline "\"ERR\"")
+  | [_; "cache-delete"; rootPath] -> (
+    Cfg.useProjectConfigCache := false;
```

Comment:
This does not seem to be used?

Reply:
It's called here: https://github.com/rescript-lang/rescript-vscode/pull/1000/files#diff-4398e3b2ab3e7e7e70a93409fa893abf68608d45a5ad1ac7faaf888dac77005dR220-R228

Reply:
Sorry I meant to say the line `Cfg.useProjectConfigCache := false` does not seem to be necessary, as none of the code below is affected by it.